src/client/components/ui/Carousel/Slide.tsx

```diff
@@ -19,6 +19,7 @@ interface SlideProps {
 	direction: string | null
 	setDirection: (direction: Direction) => void
 	isAnimated: boolean
+	imageSizes: { width: number, height: number }
```

Comment:
```suggestion
	imagesSize: { width: number, height: number }
```
"La taille des images" plutôt que "Les tailles de l'image"